### PR TITLE
Make it more clear how to deploy in README

### PR DESCRIPTION
**Pull request by matks**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Make it more clear how to deploy inside README
| Type?             | improvement
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | 
| Sponsor company   | 
| How to test?      | No need to test

### CODE REVIEW

**Comment by matks** [comment]
I would be happy to have a review from @JMorinPS or @atomiix :)

**Review by tleon — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -45,3 +45,30 @@ PHP >= 8.0.7
...
+
+* **Integration**: integration-help.prestashop-project.org
+* **Préproduction**: preprod-help.prestashop-project.org
+* **Production**: help.prestashop-project.org
+
+Those edge URLs are hosted at the Cloudflare level, proxyfying the origin GCP Storage.
+
+## Architecture
+
+![alt text](pics/architecture.png "Build & Refresh Workflow")
```
Comment (atomiix) [inline README.md:63]: @matks there is no gcp bucket for this project. The content is generated on-the-fly when a call is made and cached by cloudflare for 24h (if I recall).
There is already a documentation in the folder [.cloud](https://github.com/PrestaShop/contextual-help-api/tree/main/.cloud), maybe we just need to put it in the root folder?

Comment (matks) [reply README.md:63]: Cool to see u! Thanks for the tips

I totally missed the .cloud doc :) I think it's at the right place but it simply needs a link in the README. I expected .cloud to contain only scripts and configuration.

**Comment by matks** [comment]
Closing because of https://github.com/PrestaShop/contextual-help-api/pull/20#discussion_r1210415552

**Pull request closed without merging**